firmware_mod/www/cgi-bin/api.cgi

```diff
@@ -224,6 +224,11 @@ if [ -n "$F_action" ]; then
      $BINPATH/motor -d h -s 100 &>/dev/null &
      getReturn 1234 "success" "Motor is calibration on horizontal."
   ;;
+  motor_PTZ)                                                                                                                                                            
+     /system/sdcard/scripts/PTZpresets.sh $F_x_axis $F_y_axis                        
```

Comment:
please don't use api.cgi anymore, it contains a lot of duplicate code and will be phased out soon

Reply:
OK, I will move the code to action.cgi.